meta/main.yml

```diff
@@ -0,0 +1,22 @@
+galaxy_info:
+  author: Markus Mroch (Mr. Pi)
+  description: Configures the client config for stuvus ldap
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: CC-BY-SA
+
+  min_ansible_version: 2.4
+
+  platforms:
+    - name: Ubuntu
+      versions:
+        - xenial
+        - yakkety
+        - zesty
+        - artful
+
+  galaxy_tags:
+    - stuvus
```

Comment:
Aren't there more tags? And why is there a stuvus tag? Literally nobody will ever search that

Reply:
Please change this in ansible-skeleton if you don't agree with it.

Reply:
Okay